source/pto.rst

```diff
@@ -213,8 +213,11 @@ The same as :ref:`pto:RM3 with Hydraulic PTO`.
 
 Other PTO-Sim Tutorials
 """"""""""""""""""""""""""""""
+.. Note:: 
+	Adam:
+	The PTOs in this table are kind of all covered by and tied to the discussion above. Might consider removing to prevent confusion. Or move table to top of the PTO-Sim section and then user can find them before reading through.
 
-Other PTO-Sim tutorials that were not discussed above can be found on the `WEC-Sim Applications <https://github.com/WEC-Sim/WEC-Sim_Applications>`_ repository.
+Other PTO-Sim tutorials that were not discussed above can be found on the `WEC-Sim Applications <https://github.com/WEC-Sim/WEC-Sim_Applications>`_ repository:
 
```

Comment:
I like the idea of moving it to the top. These were always kind of an afterthought in the docs. This also relates to today's discussion about what should be in the WEC-Sim docs versus what belongs in the Applications repository README.